src/helpers/config.ts

```diff
@@ -32,3 +32,7 @@ export const PRIVATE_KEY =
   '0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80'
 
 export const DEFAULT_NETWORK_CONFIG = NETWORKS[DEFAULT_NETWORK]
+
+export function getSelectorLookupURL(selector: string) {
```

Comment:
Let's use config file for constants only, of you need a function then this goes into utils

Reply:
Changed that.